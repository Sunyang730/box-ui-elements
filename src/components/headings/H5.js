// @flow
import * as React from 'react';
import classNames from 'classnames';

import './H5.scss';

type Props = {
    children: React.Node,
    className?: string,
};

const H5 = ({ children, className }: Props) => <h5 className={classNames('bdl-H5', className)}>{children}</h5>;

export default H5;
