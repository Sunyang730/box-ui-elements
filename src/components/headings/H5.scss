@import '../../styles/variables';

h5.bdl-H5 {
    @include font-family;

    color: $bdl-gray-62;
    font-size: 11px;
    font-weight: normal;
    letter-spacing: .6px;
    line-height: 16px;
    text-transform: uppercase;
}
