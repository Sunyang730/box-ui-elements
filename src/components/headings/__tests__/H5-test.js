// @flow
import * as React from 'react';

import H5 from '../H5';

describe('components/headings/H5', () => {
    const getWrapper = (props = {}) => shallow(<H5 {...props}>foo</H5>);

    test('should render H5', () => {
        const wrapper = getWrapper({ className: 'moo' });

        expect(wrapper).toMatchSnapshot();
    });
});
