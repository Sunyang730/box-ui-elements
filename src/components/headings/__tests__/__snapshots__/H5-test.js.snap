// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/headings/H5 should render H5 1`] = `
<h5
  className="bdl-H5 moo"
>
  foo
</h5>
`;
